content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md

```diff
@@ -78,8 +78,8 @@ This product is designed to work alongside the Portenta family. Please check the
 
 ## Functional Overview
 ### Board Topology 
-### Top View
-![Topology - Top view](assets/breakoutCarrierConnectors.png)
+Front view
+![](assets/breakoutCarrierConnectors.png)
```

Comment:
Image description missing. (we may render datasheets as HTML on the website at some point)

Reply:
Fixed